relay/README.md

```diff
@@ -84,91 +65,189 @@ Scene 2:
 - QmOne tries to connect via relaying, because it shares this transport with QmTwo.
 - A lively and prolonged dialogue ensues.
 
-
 ## Addressing
 
-`/p2p-circuit` multiaddrs don't carry any meaning of their own.
-They need to encapsulate a `/p2p` address, or
-be encapsulated in a `/p2p` address, or both.
+`/p2p-circuit` multiaddrs don't carry any meaning of their own. They need to encapsulate a `/p2p` address, or be encapsulated in a `/p2p` address, or both.
+
+As with all other multiaddrs, encapsulation of different protocols determines which metaphorical tubes to connect to each other.
+
+A `/p2p-circuit` circuit address, is formated following:
+
+`/p2p-circuit[<relay peer multiaddr>]<destination peer multiaddr>`
+
+This opens the room for multiple hop relay, where the first relay is encapsulated in the seconf relay multiaddr, such as:
 
-As with all other multiaddrs, encapsulation of different protocols
-determines which metaphorical tubes to connect to each other.
+`/p2p-circuit/p2p-circuit/<first relay>/<first hop multiaddr>/<destination peer multiaddr>`
```

Comment:
Trying to reformat this example to match @lgierth clarifications `/p2p-circuit/p2p-circuit/<first relay>/<first hop multiaddr>/<destination peer multiaddr>`

I think the above is intended to read as follows:

* `<first relay>/p2-pcircuit/<second relay>/p2p-circuit/ipfs/QmDst`
    * use the first hop `<first relay>/p2-pcircuit`
         * either peer routed or using its explicit transport if specified
     * use the second hop `<second relay>/p2p-circuit`
          * either peer routed or using its explicit transport if specified
      * circuit `/ipfs/QmDst` over the <first relay> <-> <second relay> pipe